terracotta/image.py

```diff
@@ -187,6 +188,21 @@ def gamma_correction(
     return arr
 
 
+def apply_color_transform(
+        masked_data: Array,
+        color_transform: str,
+        out_dtype: type = np.uint16,
+) -> Array:
+    """Apply gamma correction to the input array and scale it to the output dtype."""
+    arr = to_math_type(masked_data)
+
+    for func in parse_operations(color_transform):
+        arr = func(arr)
+
+    arr = scale_dtype(arr, out_dtype)
```

Comment:
This should be unnecessary no? Because we scale it to `uint8` right after?

Reply:
Yes, you're right, the contrast stretch in `to_uint8` makes it unnecessary.